### ISSUE TITLE
Accessibility Issues:  Many buttons in 2FAS Android (and possibly iOS, but I cannot confirm) are unlabeled

### ISSUE THREAD

**Issue by britechguy** [twofas/2fas-android#148]
### Bug type

Other

### App version

5.4.0

### Device environment

Android 13, Xiaomi Redmi Note 11 Pro Plus 5G  (but this spans environments)

### Bug description

When someone who uses Talkback attempts to use 2FAS, they begin encountering unlabeled buttons "all over the place."  If you enable TalkBack and attempt to use 2FAS, you will encounter this.  There are more than can be mentioned and I do not know where all of them might happen to be.

### Solution

All controls, and in this case buttons in particular, should be labeled as a standard part of 2FAS code.

### Additional context

It would be wise to see if this issue extends to the iOS port and, possibly, even to the 2FAS button for the browser extensions.

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I understand that security vulnerabilities should be reported to security@2fas.com instead of on GitHub.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by heull001** [comment]
This issue still exists in app version 5.4.9. I would also be happy to receive a patch.

> It would be wise to see if this issue extends to the iOS port and, possibly, even to the 2FAS button for the browser extensions.

This issue does not exist in the iOS app.
